src/pages/run-a-node.js

```diff
@@ -0,0 +1,1011 @@
+    &:hover {
+      transition: 0.1s;
+      transform: scale(1.01);
+      img {
+        transition: 0.1s;
+        transform: scale(1.1);
+      }
+    }
+  }
+`
+
+const Width80 = styled.div`
+  line-height: 2;
+  box-sizing: border-box;
+  flex: 4;
+  ul {
+    list-style: none;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    width: 100%;
+  }
+`
+
+const Width20 = styled.div`
+  flex: 1;
+  display: flex;
+  inset: auto;
+  justify-content: center;
+  align-items: center;
+  place-items: center;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    width: 100%;
+  }
+`
+
+const Width60 = styled.div`
+  flex: 3;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    width: 100%;
+  }
+`
+
+const Width40 = styled.div`
+  flex: 2;
+  display: flex;
+  justify-content: center;
+  align-self: center;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    width: 100%;
+  }
+`
+
+const Collapse = styled(motion.div)``
+
+const Flex = styled.div`
+  display: flex;
+  gap: 2rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
+    flex-direction: column;
+  }
+`
+
+const MarginFlex = styled(Flex)`
+  margin: 3rem 0;
+`
+
+const Container = styled.div`
+  background: ${({ theme }) => theme.colors.grayBackground};
+  border: 1px solid #d1d1d1;
+  box-sizing: border-box;
+  border-radius: 5px;
+  color: ${({ theme }) => theme.colors.text};
+  padding: 0 2rem;
+`
+
+const BuildBox = styled(Container)`
+  background: ${({ theme }) => theme.colors.preBackground};
+  flex: 1;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    flex-direction: column;
+  }
+`
+
+const BuildFlex = styled(Flex)`
+  padding-bottom: 2rem;
+`
+
+const FullyLoaded = styled(Container)`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  line-height: 200%;
+  padding-bottom: 2rem;
+  p {
+    font-size: 125%;
+  }
+  code {
+    font-weight: 600;
+    line-height: 125%;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
+    button {
+      width: fit-content;
+      padding-left: 2rem;
+      padding-right: 2rem;
+    }
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    button {
+      width: 100%;
+    }
+  }
+  &:hover {
+    transition: transform 0.1s;
+    transform: scale(1.02);
+  }
+`
+
+const AdventureContainer = styled(Container)`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 2rem;
+  &:hover {
+    transition: transform 0.1s;
+    transform: scale(1.02);
+  }
+`
+
+const SvgTitle = styled.div`
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+`
+
+const ButtonContainer = styled.div`
+  display: flex;
+  gap: 1rem;
+  margin-top: 2rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    flex-direction: column;
+  }
+`
+
+const DappNodeButtonLink = styled(ResponsiveButtonLink)`
+  background-color: #30bcb2;
```

Comment:
This button doesn't pass accessibility contrast tests. Noticed you're using the Dapp node colours for this which I think is a nice touch—maybe we could tweak it slightly to make it compliant? 

![Screenshot 2022-01-07 at 23 15 15](https://user-images.githubusercontent.com/62268199/148619421-9415e87f-0ee0-4e6b-bf23-494e5deb450b.png)